.github/workflows/sync-versioning-to-cdn.yml

```diff
@@ -1,42 +0,0 @@
-name: CDN config sync
-
-on:
-  push:
-    branches:
-      - main
-    paths:
-      - build-system/global-configs/versioning.json
-
-jobs:
-  sync:
-    if: github.repository == 'ampproject/amphtml'
-    name: versioning.json
-    runs-on: ubuntu-latest
-    environment: wrangler
-
-    steps:
-      - name: Checkout Repo
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 1
-
-      - name: Install Dependencies
-        run: npm i -g @cloudflare/wrangler
-
-      - name: ⭐ Sync versioning.json to the CDN ⭐
-        run: |
-          for CHANNEL in $(jq 'keys[]' build-system/global-configs/versioning.json --raw-output)
-          do
-            RTV=$(jq ".[\"${CHANNEL}\"]" build-system/global-configs/versioning.json --raw-output)
-            if [[ "${RTV}" != "null" ]]
-            then
-              echo "Setting ${CHANNEL} to ${RTV}"
-              wrangler kv:key put "${CHANNEL}" "${RTV}" --config .github/workflows/wrangler.toml --binding RTV
```

Comment:
Can `wrangler.toml` be deleted too, or deleted later?

Reply:
It's still used by `.github/workflows/sync-client-side-experiments-config-to-cdn.yml` so I'll delete it with the PR that removes the client-side experiments from this repo